WalletWasabi/WabiSabi/Models/MultipartyTransaction/SigningState.cs

```diff
@@ -89,5 +99,10 @@ public Transaction CreateTransaction()
 
 			return tx;
 		}
+
+		private IEnumerable<Coin> GetSortedInputs() =>
+			Inputs
+				.OrderByDescending(x => x.Amount)
+				.ThenBy(x => x.Outpoint.ToBytes(), ByteArrayComparer.Comparer);
```

Comment:
Since C#'s `Enumerable.OrderBy` is stable, this is not required.

On slack I suggested this to still do this because it is a bit more explicit and helps readability, but I didn't realize a new comparer for ByteArrays would be required, so I think I was mistaken.

Feel free to remove if you prefer it without.

Reply:
I prefer to make this explicit.